paper.tex

```diff
@@ -881,6 +817,7 @@ \section{Extended data and figures}
         form. (C) A visual representation of the model using \texttt{demesdraw}.
         The YAML descriptions in (A) and (B) correspond exactly to a JSON description,
         but are much more human-readable.
+        \aprcomment{Need figure panel labels.}
```

Comment:
Do we also need to delete these lines?

        \begin{tcolorbox}
            \includegraphics[width=\textwidth]{fig/IM}
        \end{tcolorbox}
        
When I try and compile the .tex file on my machine then it throws an error as this figure is missing; I take it that it's been superseded by 'models/IM.yaml'?

Reply:
That's strange, as it should be generated by the Makefile if it is missing. Can you paste the full error message when you do `make`?

Reply:
Ah, I get it now, I was trying to create the PDF using latex rather than the makefile. Now it works fine!